### DISCUSSION TITLE
SubStack integration

### DISCUSSION THREAD

**Discussion by lukejans** [dailydotdev/daily#677] [Feature Requests]
I feel like it would be an interesting feature to have 2 search bars. one to search articles on daily dev and one to search on substack for the more specific needs.

tell me if this is stupid and I should just go to substack every time.

**Comment by idoshamun** [comment]
Hi @lukejans,

This is an interesting suggestion, but it's out of our scope at the moment

**Reply by lukejans** [reply]
Just realizing that I meant to say stackOverflow... but subStack could be interesting.
